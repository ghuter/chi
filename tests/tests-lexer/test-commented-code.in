// This is an amazing function:
amazing :: fun () ^u8 {
	msg :: "I'm amazing o_o"; // this is my message for the world !
	return msg;
}
