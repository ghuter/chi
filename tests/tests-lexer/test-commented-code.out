NEWLINE(6)
IDENTIFIER(5)
<amazing>
SPC(8)
COLON(9)
COLON(9)
SPC(8)
FUN(64)
SPC(8)
LPAREN(12)
RPAREN(13)
SPC(8)
BXOR(32)
IDENTIFIER(5)
<u8>
SPC(8)
LBRACES(16)
NEWLINE(6)
SPC(8)
IDENTIFIER(5)
<msg>
SPC(8)
COLON(9)
COLON(9)
SPC(8)
LITERAL(4)
"I'm amazing o_o"
SEMICOLON(7)
SPC(8)
NEWLINE(6)
SPC(8)
RETURN(59)
SPC(8)
IDENTIFIER(5)
<msg>
SEMICOLON(7)
NEWLINE(6)
RBRACES(17)
EOI(1)
